src/components/pages/Step5.vue

```diff
@@ -229,12 +229,70 @@
                         </v-card-actions>
                     </v-card>
                 </v-dialog>
+                
 
+                <!-- Archive session -->
+                <v-btn small class="mt-4 w-100" @click="showArchiveDialog = true">
+                    Download data
+                </v-btn>
+                <v-dialog
+                    v-model="showArchiveDialog"
+                    max-width="800">
+                    <v-card>
+                        <v-card-text class="pt-4">
+                            <v-row class="m-0">
+                            <v-col cols="2">
+                                <v-icon x-large color="green">mdi-download</v-icon>
+                            </v-col>
+                            <v-col cols="10">
+                                <p v-if="isArchiveInProgress & !isArchiveDone">
+                                    <v-progress-circular  indeterminate class="mr-2" color="grey" size="14" width="2" />
+                                    Download in progress...
+                                </p>
+                                <p v-if="!(isArchiveInProgress || isArchiveDone)">
+                                    Do you want to download all data associated to the
+                                    session <code>{{session.id}}</code>?
+                                    (This includes every trial associated to it, and can take several minutes).
+                                </p>
+                                <p v-if="isArchiveDone">
+                                    The archive has been generated successfully!
```

Comment:
Rephrase: The archive has been successfully generated. Click on data.zip to download.